Detectors/Upgrades/ALICE3/EC0/simulation/src/Detector.cxx

```diff
@@ -0,0 +1,557 @@
+  TString path = Form("%s/%s%d_1", parent.Data(), GeometryTGeo::getEC0LayerPattern(), lr);
+
+  TString sname = GeometryTGeo::composeSymNameLayer(lr);
+
+  LOG(DEBUG) << "Add " << sname << " <-> " << path;
+
+  if (!gGeoManager->SetAlignableEntry(sname, path.Data())) {
+    LOG(FATAL) << "Unable to set alignable entry ! " << sname << " : " << path;
+  }
+  addAlignableVolumesChip(lr, path, lastUID);
+  return;
+}
+
+//_________________________________________________________________________________________________
+void Detector::addAlignableVolumesChip(int lr, TString& parent, Int_t& lastUID) const
+{
+
+  TString path = Form("%s/%s%d_1", parent.Data(), GeometryTGeo::getEC0ChipPattern(), lr);
+
+  TString sname = GeometryTGeo::composeSymNameChip(lr);
+  Int_t modUID = chipVolUID(lastUID++);
+
+  LOG(DEBUG) << "Add " << sname << " <-> " << path;
+
+  if (!gGeoManager->SetAlignableEntry(sname, path.Data(), modUID)) {
+    LOG(FATAL) << "Unable to set alignable entry ! " << sname << " : " << path;
+  }
+  //addAlignableVolumesSensor(lr, path, lastUID);
+  return;
+}
+
+//_________________________________________________________________________________________________
+void Detector::addAlignableVolumesSensor(int lr, TString& parent, Int_t& lastUID) const
+{
+
+  TString path = Form("%s/%s%d_1", parent.Data(), GeometryTGeo::getEC0SensorPattern(), lr);
+
+  TString sname = GeometryTGeo::composeSymNameSensor(lr);
+  Int_t modUID = chipVolUID(lastUID++);
+
+  LOG(DEBUG) << "Add " << sname << " <-> " << path;
+
+  if (!gGeoManager->SetAlignableEntry(sname, path.Data(), modUID)) {
+    LOG(FATAL) << "Unable to set alignable entry ! " << sname << " : " << path;
+  }
+
+  return;
+}
+
+//_________________________________________________________________________________________________
+void Detector::defineSensitiveVolumes()
+{
+  TGeoManager* geoManager = gGeoManager;
+  TGeoVolume* v;
+
+  TString volumeName;
+
+  // The names of the EC0 sensitive volumes have the format: EC0Sensor(0...sNumberLayers-1)
+  for (Int_t j = 0; j < mNumberOfLayers; j++) {
+    volumeName = o2::ec0::GeometryTGeo::getEC0SensorPattern() + std::to_string(j);
+    v = geoManager->GetVolume(volumeName.Data());
+    LOG(INFO) << "Adding EC0 Sensitive Volume => " << v->GetName() << std::endl;
+    AddSensitiveVolume(v);
+  }
+}
+
+//_________________________________________________________________________________________________
+Hit* Detector::addHit(int trackID, int detID, const TVector3& startPos, const TVector3& endPos,
+                      const TVector3& startMom, double startE, double endTime, double eLoss, unsigned char startStatus,
+                      unsigned char endStatus)
+{
+  mHits->emplace_back(trackID, detID, startPos, endPos, startMom, startE, endTime, eLoss, startStatus, endStatus);
+  return &(mHits->back());
+}
+
+//_________________________________________________________________________________________________
+void Detector::Print(std::ostream* os) const
```

Comment:
I wonder if these methods will be ever used. I am planning to remove them from the ITS3/TRK implementation.
They are quite obsolete. 
Do you think it there is/will be a use case that will make useful to propgate their definition trhougout the years?